controllers/search_controller.rb

```diff
@@ -8,6 +8,14 @@ class SearchController < ApplicationController
       process_search
     end
 
+    get '/content' do
+      process_search
+    end
+
+    get '/metadata' do
+      process_search
```

Comment:
here use https://github.com/ontoportal-lirmm/ontologies_api/blob/master/controllers/search_controller.rb#L184, it can search in the metadata of ontologies.